packages/node/src/transports/http.ts

```diff
@@ -23,6 +24,30 @@ export class HTTPTransport extends BaseTransport {
     if (!this.module) {
       throw new SentryError('No module available in HTTPTransport');
     }
-    return this._sendWithModule(this.module, event);
+    return this._sendWithModule(this.module, eventToSentryRequest(event, this._api), event);
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public sendSession(session: Session): PromiseLike<Response> {
+    if (!this.module) {
+      throw new SentryError('No module available in HTTPTransport');
+    }
+    return this._sendWithModule(this.module, sessionToSentryRequest(session, this._api), session);
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public sendSessions(aggregatedSessions: AggregatedSessions): PromiseLike<Response> {
```

Comment:
Would it make sense to reserve `sendSessions` for `Session[]` and make this `sendSessionAggregate` (pro: name sorts and groups together with `sendSession` in docs, code completion tooltip,`grep`, etc) or `sendAggregatedSessions` (pro: name matches the input type, just like `sendEvent` and `sendSession` do)?

We could get best of both names if `AggregatedSessions` was called `SessionAggregate`. Is there a standard name for this already?